csharp/Platform.Collections/Arrays/GenericArrayExtensions.cs

```diff
@@ -127,44 +135,94 @@ public static IList<T> ShiftRight<T>(this T[] array, long shift)
         }
 
         /// <summary>
-        /// <para>One of the array values with index on variable position++ type int is passed to the element variable.</para>
-        /// <para>Одно из значений массива с индексом переменной position++ типа int назначается в переменную element.</para>
+        /// <para>Adding in array the passed element at the specified position.</para>
+        /// <para>Добавляет в массив переданный элемент на указанную позицию.</para>
         /// </summary>
-        /// <param name="array"><para>An array whose specific value will be assigned to the element variable.</para><para>Массив, определенное значений которого присваивается переменной element</para></param>
-        /// <param name="position"><para>Reference to a position in an array of int type.</para><para>Ссылка на позицию в массиве типа int.</para></param>
-        /// <param name="element"><para>The variable which needs to be assigned a specific value from the array.</para><para>Переменная, которой нужно присвоить определенное значение из массива.</para></param>
         /// <typeparam name="T"><para>Array elements type.</para>Тип элементов массива.<para></typeparam>
+        /// <param name="array"><para>The array to add the element to.</para><para>Массив в который необходимо добавить элемент.</para></param>
+        /// <param name="position"><para>A reference to the position of type int where the element will be added.</para><para>Ссылка на позицию типа int, в которую будет добавлен элемент.</para></param>
+        /// <param name="element"><para>The value of variable to add to the array.</para><para>Переменная, значение которой нужно добавить в массив</para></param>
```

Comment:
`The element to add to the array.` Fix Russian version too (don`t forget to add  the dot at the end).